modules/ocf_stats/manifests/munin.pp

```diff
@@ -1,7 +1,6 @@
 # munin master config
 class ocf_stats::munin {
   include ocf::firewall::allow_web
-  include ocf_ssl::default_bundle
```

Comment:
Isn't this still needed? I don't see how else the certs are getting provided (I'll be changing this soon to use Let's Encrypt though, but that's besides the point)

Reply:
It's already included in the main `ocf_stats` class.